Source/WebKit/Shared/Extensions/_WKWebExtensionLocalization.mm

```diff
@@ -0,0 +1,375 @@
+
+    while (positionalPlaceholderMatch) {
+        NSInteger position = [string substringWithRange:[positionalPlaceholderMatch rangeAtIndex:2]].integerValue;
+
+        NSString *replacement;
+        if (position > 0 && position <= placeholderCount)
+            replacement = placeholderValues[position - 1];
+        else
+            replacement = @"";
+
+        NSString *stringToReplace = [string substringWithRange:[positionalPlaceholderMatch rangeAtIndex:1]];
+
+        string = [string stringByReplacingOccurrencesOfString:stringToReplace withString:replacement options:NSCaseInsensitiveSearch range:NSMakeRange(0, string.length)];
+        positionalPlaceholderMatch = [positionalPlaceholderExpression firstMatchInString:string options:0 range:NSMakeRange(0, string.length)];
+    }
+
+    return string;
+}
+
+// MARK: NSSecureCoding
+
+- (void)encodeWithCoder:(NSCoder *)coder
+{
+    [coder encodeObject:_localizationDictionary forKey:localizationDictionaryCodingKey];
+    [coder encodeObject:_localeString forKey:localeStringDictionaryCodingKey];
+    [coder encodeObject:_locale forKey:localeCodingKey];
+
+    if (_uniqueIdentifier)
+        [coder encodeObject:_uniqueIdentifier forKey:uniqueIdentifierCodingKey];
+}
+
+- (instancetype)initWithCoder:(NSCoder *)coder
+{
+    if (!(self = [self init]))
+        return nil;
+
+    _localizationDictionary = [coder decodeObjectOfClasses:[NSSet setWithObjects:NSDictionary.class, NSString.class, NSArray.class, nil] forKey:localizationDictionaryCodingKey];
+    _localeString = [coder decodeObjectOfClass:NSString.class forKey:localeStringDictionaryCodingKey];
+    _locale = [coder decodeObjectOfClass:NSLocale.class forKey:localeCodingKey];
+    _uniqueIdentifier = [coder decodeObjectOfClass:NSString.class forKey:uniqueIdentifierCodingKey];
+
+    return self;
+}
+#else
+
+- (instancetype)initWithBundleURL:(NSURL *)bundleURL defaultLocale:(NSString *)defaultLocaleString uniqueIdentifier:(NSString *)uniqueIdentifier
+{
+    return [self initWithRegionalLocalization:nil languageLocalization:nil defaultLocalization:nil withBestLocale:nil uniqueIdentifier:uniqueIdentifier];
+}
+
+- (instancetype)initWithRegionalLocalization:(LocalizationDictionary *)regionalLocalization languageLocalization:(LocalizationDictionary *)languageLocalization defaultLocalization:(LocalizationDictionary *)defaultLocalization withBestLocale:(NSString *)localeString uniqueIdentifier:(NSString *)uniqueIdentifier
+{
+    return nil;
+}
+
+- (NSDictionary<NSString *, id> *)localizedDictionaryForDictionary:(NSDictionary<NSString *, id> *)dictionary
+{
+    return nil;
+}
+
+- (NSString *)localizedStringForKey:(NSString *)key withPlaceholders:(NSArray<NSString *> *)placeholders
+{
+    return nil;
+}
+
+- (NSString *)localizedStringForString:(NSString *)string
+{
+    return nil;
+}
+
+- (instancetype)initWithCoder:(NSCoder *)coder
+{
+    return nil;
```

Comment:
Try just adding: `return [self initWithRegionalLocalization:nil languageLocalization:nil defaultLocalization:nil withBestLocale:nil uniqueIdentifier:uniqueIdentifier];` here to fix the build.